### ISSUE TITLE
Access to in-memory databases

### ISSUE THREAD

**Issue by hclivess** [sqlitebrowser/sqlitebrowser#1369]
Can access to in-memory databases with shared cache be added?

https://www.sqlite.org/inmemorydb.html

**Comment by hclivess** [comment]
Yes, same thing. Maybe we should ping @glawrence about it

**Comment by glawrence** [comment]
@hclivess not sure why you are pinging me as I am not sure what I can do to help, sorry

**Comment by hclivess** [comment]
Sorry, I thought you planned to add this in 2016, pehaps I misread in #335

**Comment by glawrence** [comment]
Right, sorry, I certainly didn't intend to mis-lead anyone but for clarity, I am not planning to add support for in memory databases.

I have read your link above and I do not see what DB4S can do. My understanding is we could add in-memory support but the database would only ever exist in DB4S and not be available to anything else. It seems clear that DB4S could not access an in-memory database from another process. I get there is a shared cache but I fail to see what that would do for us. Do you have a specific use case or example in mind?

**Comment by justinclift** [comment]
Cool.  No worries at all @hclivess. :smile:
